omnigibson/robots/vx300s.py

```diff
@@ -115,7 +115,7 @@ def _create_discrete_action_space(self):
         raise ValueError("VX300S does not support discrete actions!")
 
     @property
-    def controller_order(self):
+    def _raw_controller_order(self):
```

Comment:
General question: it seems to me that this could easily be implemented in the manipulation/articulatedtrunk/locomotion etc robot classes since it's always exactly the same thing for every robot? (e.g. each class calls super and appends its extra element).

Reply:
Same for default_controllers.

Reply:
Yeah this is a great point. The only downside is that it then comes down to the arbitrary superclass definition order defined by each Robot class (e.g.: R1), so it might not be consistent across different robots